src/vs/workbench/services/extensionManagement/browser/webExtensionsScannerService.ts

```diff
@@ -621,11 +641,12 @@ export class WebExtensionsScannerService extends Disposable implements IWebExten
 		return [];
 	}
 
-	private async translateManifest(manifest: IExtensionManifest, nlsURL: URI): Promise<IExtensionManifest> {
+	private async translateManifest(manifest: IExtensionManifest, nlsURL: URI, fallbackNlsURL: URI): Promise<IExtensionManifest> {
 		try {
 			const content = await this.extensionResourceLoaderService.readExtensionResource(nlsURL);
+			const fallbackContent = await this.extensionResourceLoaderService.readExtensionResource(fallbackNlsURL);
```

Comment:
We need both because the language specific file isn't guaranteed to have all the strings in it. So it's like... first we see if there's a string in the localized file, if not we see if there's a string in the english file.